File: 백준/Bronze/2576. 홀수/홀수.py
import sys
input = sys.stdin.readline

nums = [int(input()) for _ in range(7)]
for num in nums.copy():
    if num % 2 == 0:
        nums.remove(num)

print(f'{sum(nums)}\n{min(nums)}' if len(nums) != 0 else -1)

File: 백준/Bronze/2576. 홀수/홀수.cc
#include <iostream>

using namespace std;

int main(){
    int min = 100, sum = 0, num;
    for (int i = 0; i < 7; i++){
        cin >> num;

        if (num % 2){
            sum += num;

            if (num < min)
                min = num;
        }
    }

    if (!sum)
        cout << -1 << endl;
    else
        cout << sum << endl << min << endl;

    return 0;
}